packages/in_app_purchase/in_app_purchase_storekit/shared/Classes/FIAPaymentQueueHandler.m

```diff
@@ -160,15 +160,23 @@ - (void)restoreTransactions:(nullable NSString *)applicationName {
 }
 
 - (void)presentCodeRedemptionSheet {
```

Comment:
Shouldn't these APIs be marked iOS-only, so that we don't call them at all on macOS? We don't want to be constantly logging in normal usage.